### PR TITLE
DEVDOCS-4719: [migrate] Checkout SDK, migrate syntax

### PR DESCRIPTION
**Pull request by bc-traciporter**
# [DEVDOCS-4719]

## What changed?
Updated code blocks and callouts in Checkout SDK Tutorial

## Anything else?
Related PRs, salient notes, etc

[DEVDOCS-4719]: https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4719?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.mdx [hunk 1]
```diff
@@ -23,9 +24,10 @@ If you have added any new subdirectories or files to your base theme, verify tha
 * Set newly added directories to permission `755` (`drwxr-xr-x`).
 * Set newly added files to permission `644` (`rw-r--r--`).
 
-<!-- theme: warning -->
-> #### Writable permissions are required
-> Without these writable permissions, bundling your theme will fail, blocking its upload to BigCommerce.
+<Callout type="warning" emoji="⚠️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.mdx:27]:
```suggestion
<Callout type="warning">

```

File: docs/stencil-docs/deploying-a-theme/bundling-and-pushing.mdx [hunk 2]
```diff
@@ -137,10 +139,11 @@ Use your arrow keys to move the selection caret to each theme/version that you w
 If you are certain of your selections, you can press **Enter** to delete the themes.
 
 
-<!-- theme: danger -->
-> #### Select carefully – no confirmation
-> Once you press `Enter`, the selected themes will be deleted immediately, with no further confirmation.
-> If you have any doubts – especially about deleting multiple themes/versions – it is safest to delete them through the control panel GUI. This GUI allows you to compare uploaded versions and to inspect their metadata.
+<Callout type="warning" emoji="⚠️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/bundling-and-pushing.mdx:142]:
```suggestion
<Callout type="warning">

```

File: docs/stencil-docs/deploying-a-theme/checking-a-themes-size.mdx [hunk 3]
```diff
@@ -41,9 +41,10 @@ On your local machine, move the large static assets to a location where the `ste
 
 Run the `stencil bundle` command from inside your streamlined `theme‑name` directory. Once your resulting ZIP file is 50 MB or smaller, you are ready to upload it to BigCommerce. For more information, see [Bundling and Pushing a Theme](/stencil-docs/deploying-a-theme/bundling-and-pushing).
 
-<!-- theme: warning -->
-> #### URL references to assets
-> Both in production and locally, subdirectories of your theme's `assets` directory, such as `img`,`js`, and `fonts`, are parallel to its `scss` subdirectory. Within your CSS, path references to such assets should reflect this parallel relationship, for example: `../img/test.jpg`.
+<Callout type="warning" emoji="⚠️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/checking-a-themes-size.mdx:44]:
```suggestion
<Callout type="warning">

```

File: docs/stencil-docs/deploying-a-theme/checking-a-themes-size.mdx [hunk 4]
```diff
@@ -76,8 +77,9 @@ As you develop your theme, reference each of these assets using Stencil's `{{cdn
 <img src="{{cdn "webdav:img/image.jpg"}}">
 ```
 
-<!-- theme: info -->
-> The presumed WebDAV root directory is `content`. In this example, the `image.jpg` file had been uploaded to the WebDAV `content` directory. The presumed local directory for other CDNs is `assets`, so you can omit that path when referencing its contained files or subdirectories.
+<Callout type="info" emoji="ℹ️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/checking-a-themes-size.mdx:80]:
```suggestion
<Callout type="info">

```

File: docs/stencil-docs/deploying-a-theme/naming-your-theme.mdx [hunk 5]
```diff
@@ -108,10 +108,11 @@ The following example shows how to set the body font of a theme variation to a s
 
 For details on preparing screenshots that enable merchants to preview each variation, and to preview your theme as a whole in the BigCommerce control panel and Theme Marketplace, see [Preparing Thumbnail Images](/stencil-docs/deploying-a-theme/preparing-thumbnail-images).
 
-<!-- theme: info -->
-> #### Using Stencil CLI to change variations
-> To locally test/debug a specific variation of your theme, launch it by invoking the Stencil CLI's `stencil start -v` switch. Enter the following code into the terminal, replacing `variation-name` with your theme's variation name as listed in `config.json`:
-> `stencil start -v variation-name`
+<Callout type="info" emoji="ℹ️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/naming-your-theme.mdx:111]:
```suggestion
<Callout type="info">

```

File: docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx [hunk 6]
```diff
@@ -8,7 +8,7 @@ There are two ways to import images. The steps below describe the quickest and e
 2. Add images to the `/content` directory.
 3. Upload the images using a WebDAV client. You can access the images within the theme using the `cdn` Handlebars helper as follows:
 
-```handlebars title="cdn helper"
+```handlebars filename="cdn helper"
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx:11]:
```suggestion
```handlebars filename="cdn helper" copy
```

File: docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx [hunk 7]
```diff
@@ -57,9 +57,10 @@ We recommend that you use Lighthouse to focus on the following metrics for optim
 ### Largest Contentful Paint
 Largest Contentful Paint (LCP) is the metric that reports the time it takes to display the largest image on the screen. This metric is important because page speed ensures visitors receive the best performance and presentation of the site.
 
-<!-- theme: info -->
-> #### Good score
-> The optimal LCP score is 2.5 seconds or faster. 
+<Callout type="info" emoji="ℹ️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx:60]:
```suggestion
<Callout type="info">
```

File: docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx [hunk 8]
```diff
@@ -70,9 +71,10 @@ To improve LCP:
 ### First Input Delay
 First Input Delay (FID) is the metric that measures the delay users experience when interacting with a page. It is the time it takes for the site to respond when a user clicks a link, selects a button, taps on the screen, etc.
 
-<!-- theme: info -->
-> #### Good score
-> The optimal FID score is 100 milliseconds or less. 
+<Callout type="info" emoji="ℹ️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx:74]:
```suggestion
<Callout type="info">
```

File: docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx [hunk 9]
```diff
@@ -87,9 +89,10 @@ A good TBT score is 300 milliseconds or less.
 
 Cumulative Layout Shift (CLS) is the metric that measures visual stability. This metric measures how often users experience unexpected layout shifts. A layout shift is when a visible element changes its position from one rendered frame to the next. For example, a layout shift can occur when a user views a page and a video moves on top of the home carousel. These unexpected movements can result in an unpleasant experience for the user.
 
-<!-- theme: info -->
-> #### Good score
-> The optimal CLS score is 0.1 seconds or less. 
+<Callout type="info" emoji="ℹ️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/deploying-a-theme/theme-best-practices.mdx:92]:
```suggestion
<Callout type="info">
```

File: docs/stencil-docs/developing-further/catalog-price-object.mdx [hunk 10]
```diff
@@ -212,9 +212,10 @@ In this example:
 | `with_tax` | This property represents the `without_tax` value plus a 10% tax markup.|
 | `rrp_with_tax` | This property represents the `rrp_without_tax` value plus a 10% tax markup.|
 
-<!-- theme: warning -->
-> #### Tax display configurations
-> The same properties and values are returned for **Including tax** and **Including and excluding tax** display settings. These properties and values are not available for a control panel setting of **Excluding tax**.
+<Callout type="warning" emoji="⚠️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/developing-further/catalog-price-object.mdx:215]:
```suggestion
<Callout type="warning">

```

File: docs/stencil-docs/developing-further/customizing-invoices.mdx [hunk 11]
```diff
@@ -30,9 +30,10 @@ The invoice that is generated by clicking **Print Invoice** relies on template f
 2. Open the file in the text editor of your choice and edit the HTML file to match your desired customization.
 3. Use WebDAV to upload your customized HTML file to WebDAV's /template/ folder. We recommend CyberDuck, the freeware [WebDAV](https://support.bigcommerce.com/s/article/File-Access-WebDAV#webdav-client) client, available for Windows and Mac OS. Your store's control panel offers a downloadable connection file pre-filled with most store credentials.
 
-<!-- theme: info -->
-> #### Note
-> * For a list of available variables, see [Blueprint Email Templates](/legacy/blueprint-themes/blueprint-email-templates).
+<Callout type="info" emoji="ℹ️">
```
Comment (bc-tgomez) [inline docs/stencil-docs/developing-further/customizing-invoices.mdx:33]:
```suggestion
<Callout type="info">

```

**Review by bc-tgomez — APPROVED** [review]
I added some suggestions around the `showLineNumbers` and suggested to remove the emojis since the other files I reviewed didn't have them. Lots of great work with these files!

**Review by bc-traciporter — COMMENTED** [review]
All files look good. I had to remove "import { Callout } from 'nextra-theme-docs';" from event-hooks.mdx.

**Review by slsriehl — APPROVED** [review]
keep an eye on these articles in the preview next week.

**Pull request merged**